packages/docs/src/styleguide.md

```diff
@@ -1,16 +1,18 @@
-Blueprint is a React UI toolkit for the web.
+Blueprint is a React-based UI toolkit for the web.
 
-Development occurs in [palantir/blueprint](https://github.com/palantir/blueprint).
-Please file issues on the [GitHub issue tracker](https://github.com/palantir/blueprint/issues).
+Development and issue tracking occurs in [github.com/palantir/blueprint](https://github.com/palantir/blueprint).
 
-Releases are tagged and documented [here](https://github.com/palantir/blueprint/releases)
+Use the [__blueprintjs__ tag on Stack Overflow](http://stackoverflow.com/questions/tagged/blueprintjs)
+for support requests.
+
+Releases are tagged and documented [here on Github](https://github.com/palantir/blueprint/releases)
 (more info about releases [on the wiki](https://github.com/palantir/blueprint/wiki/Releases-&-Artifacts)).
```

Comment:
It exists internally, I looked at it, seems pretty unnecessary and out-of-date. Let's remove this now?